packages/commcare/test/index.js

```diff
@@ -679,11 +678,131 @@ describe('HTTP wrappers', () => {
       },
     };
 
-    const { data, response } = await 
-      request('GET', '/a/asri/api/v0.5/case')
-    (state);
+    const { data, response } = await request(
+      'GET',
+      '/a/asri/api/v0.5/case'
+    )(state);
 
     expect(data).to.haveOwnProperty('case_id');
     expect(response.statusCode).to.equal(200);
   });
-});
\ No newline at end of file
+});
+
+describe('Bulk', () => {
+  it('should submit JSON as an xls file', async () => {
```

Comment:
Not really submitting an xls file, That's what the adaptor is doing being the scene. Maybe something very specific like, `should bulk upload case-data`?